torch/lib/c10d/test/ProcessGroupNCCLTest.cpp

```diff
@@ -414,6 +414,11 @@ void testReduceScatter(const std::string& path, int rank, int size) {
 }
 
 int main(int argc, char** argv) {
+  if (!at::cuda::is_available()) {
+    // skip the test
+    std::cout << "CUDA not available, skipping test" << std::endl;
```

Comment:
Is it possible to use glog here?

Reply:
It doesn't seem like this is used, but looks like we can do `LOG(INFO) << message` which seems to be the pattern in the codebase.

Reply:
Changed it to use `LOG(INFO) << ....`